### DISCUSSION TITLE
Sunrise/Sunset : Heliocron

### DISCUSSION THREAD

**Discussion by Fish31520** [mcguirepr89/BirdNET-Pi#782] [Q&A]
Hi,

I'm trying to setup a second Birdnet/RPi but only for night birds.

So I'm using cron/crontab and I've discovered Heliocron which could be the best option for me.
But so far I'm not able to make it work.

My goal is for Birdnet to start at sunset and to shutdown at sunrise.

Is anybody here using Heliocron and could help me with it ?

Thanks !

**Comment by norfolk-fr** [comment]
Hi Fish,
I've been using sunwait [https://github.com/risacher/sunwait] for my chiken home to open / close the door at sunset / sunrise.
I think Heliocron should also do the trick. If you look at the Heliocron page at Github, you'll find this example : 
```
0 2 * * * heliocron --latitude 51.4769 --longitude -0.0005 wait --event sunrise --offset -00:30 \
&& turn-on-lights.sh
```

You have to fine tune the crontab entry to fit your lat and log coordinates and then replace the last command after && with something like : ~/BirdNet-Pi/scripts/stop_core_services.sh
Add another line for sunset with ~/BirdNet-Pi/scripts/restart_services.sh

This should stop BirdNet-Pi services at sunrise and start BirdNet-Pi services at sunset

Have a nice day
Alban

**Reply by Fish31520** [reply]
@norfolk-fr Thanks a lot for your help.
I'm away from my RPi but as soon as I'll get back, I'll try that.
Is Sunwait working well ?
